cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -212,7 +220,7 @@ message NodeLocation {
 }
 message DataVolume {
     required string path = 1;
-    required int64 sizeMb = 2;
+    optional int64 sizeMb = 2;
```

Comment:
This needs to be optional since we're not able to constrain or verify the size of the data volume since we're not using persistent volumes.